@AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/electrical/fn_electricalBattery.sqf

```diff
@@ -45,7 +45,7 @@ if (_battBusOn && _acBusOn) then {
 if (_battBusOn && _acBusOn) then {
     _battPower = [_battPower, 1.0, _deltaTime] call BIS_fnc_lerp;
 };
-_heli setVariable ["fza_systems_battPower_pct", _battPower];
+[_heli, "fza_systems_battPower_pct", _battPower] call fza_fnc_updateNetworkGlobal;
```

Comment:
Isn't this going to setVariable every frame?

Reply:
only if something changes like system damage, or battery state switch